src/components/Dashboard/StatusList.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+
+import * as status from '../../services/status';
+
+import Panel from '../commons/Panel';
+import Spinner from '../commons/Spinner';
+
+const STATUS_UP_CLASS = 'status-up';
```

Comment:
Can these constants be exported from the service `'../../services/status'`? All UI agnostic code should go to services.